### PR TITLE
Get Reviews by Student

### PR DESCRIPTION
**Pull request by juliejeong**
### Summary 

This PR added an endpoint [getReviewsByStudentId] in Profile.ts that returns the list of Review objects for the given student's netID. 

### Test Plan 

Tested locally. Since there is a discrepancy between the test review objects and review objects generated by the database, the number of returned reviews is compared for convenience.

### CODE REVIEW

File: server/endpoints/Profile.test.ts [hunk 1]
```diff
@@ -140,7 +176,7 @@ const mockVerificationTicket = jest.spyOn(Auth, 'getVerificationTicket')
   .mockImplementation(async (token?: string) => validTokenPayload);
 
 afterAll(async () => {
-  await mockVerificationTicket.mockRestore();
+  mockVerificationTicket.mockRestore();
```
Comment (kungpaogao) [inline server/endpoints/Profile.test.ts:179]: why did we remove `await` here?

Comment (AkashAryal) [reply server/endpoints/Profile.test.ts]: It looks like the method is returns void so we didnt need to have await there

**Review by AkashAryal — CHANGES_REQUESTED** [review]
Looks good! A small thing so that we don't have repeated code: could you create a function that returns the review objects when given a netid and use that here and in the other endpoint function? That way we don't have duplicate code

**Comment by juliejeong** [comment]
> Looks good! A small thing so that we don't have repeated code: could you create a function that returns the review objects when given a netid and use that here and in the other endpoint function? That way we don't have duplicate code

changed!

**Review by AkashAryal — APPROVED** [review]
Looks good!! Thanks for the quick update Julie <3

**Review by michelleli01 — COMMENTED** [review]
The code is easy to read and the endpoint does get all the reviews by a particular student based on netId. tThe test cases are thorough, well written, and all pass when I run it locally.

**Pull request merged**